app/classifier/tasks/shortcut-editor.spec.js

```diff
@@ -0,0 +1,109 @@
+/* eslint prefer-arrow-callback: 0, func-names: 0, 'react/jsx-boolean-value': ['error', 'always'], 'react/jsx-filename-extension': 0 */
+/* global describe, it, beforeEach */
+import { mount } from 'enzyme';
+import React from 'react';
+import assert from 'assert';
+import sinon from 'sinon';
+import ShortcutEditor from './shortcut-editor';
+
+const task = {
+  question: 'What is it',
+  type: 'single',
+  unlinkedTask: 'T1'
+};
+
+const workflow = {
+  tasks: {
+    T1: {
+      answers: [
+        { label: 'Nothing Here' },
+        { label: 'Fire' }
+      ],
+      type: 'shortcut'
+    }
+  }
+};
+
+const emptyWorkflow = {
+  tasks: {
+    T0: {
+      answers: [],
+      type: 'single'
+    }
+  }
+};
+
+const emptyTask = {
+  question: 'What is it',
+  type: 'single'
+};
+
+describe('ShortcutEditor', function () {
+  describe('shortcutRendering', function () {
+    let wrapper;
+
+    beforeEach(function () {
```

Comment:
I think this can just be a `before()` call. The set of tests in this describe block aren't interacting with the component, just checking the expected render.